### PR TITLE
tkt-70309: ix-proftpd to middlewared

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

File: src/middlewared/middlewared/etc_files/local/proftpd.conf [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+            % if ftp['anonuserdlbw']:
+                TransferRate RETR ${ftp['anonuserdlbw']}
+            % endif
+            <Limit LOGIN>
+                AllowAll
+            </Limit>
+        </Anonymous>
+    % endif
+% elif ftp['onlylocal']:
+    <Limit LOGIN>
```
Comment (sonicaj) [inline src/middlewared/middlewared/etc_files/local/proftpd.conf]: Is the indentation intended here and above as well ? Looking at the ix.rc.d script, I don't think that's the case

File: src/middlewared/middlewared/etc_files/local/proftpd.conf [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+            % endif
+            % if ftp['anonuserdlbw']:
+                TransferRate RETR ${ftp['anonuserdlbw']}
+            % endif
+            <Limit LOGIN>
+                AllowAll
+            </Limit>
+        </Anonymous>
+    % endif
+% elif ftp['onlylocal']:
```
Comment (sonicaj) [inline src/middlewared/middlewared/etc_files/local/proftpd.conf]: This also isn't an if else in the script. I think this makes sense that it should be but just pointing out to be sure that we are good wrt this

File: src/middlewared/middlewared/etc_files/local/proftpd.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import os
+
+
+async def render(service, middleware):
```
Comment (sonicaj) [inline src/middlewared/middlewared/etc_files/local/proftpd.py]: We are interacting with the filesystem, do you think we should perhaps consider a synchronous method ?

File: src/middlewared/middlewared/etc_files/local/proftpd.conf (hunk 1 above)
Comment (sonicaj) [reply src/middlewared/middlewared/etc_files/local/proftpd.conf]: It is, agreed. That resolves this

File: src/middlewared/middlewared/etc_files/local/proftpd.conf (hunk 2 above)
Comment (themylogin) [reply src/middlewared/middlewared/etc_files/local/proftpd.conf]: Actually you are right, I've changed the logic, thank you, I've rolled it back

Comment (sonicaj) [reply src/middlewared/middlewared/etc_files/local/proftpd.conf]: Thank you :)

**Pull request merged**